packages/contracts-bedrock/src/dispute/FaultDisputeGameN.sol

```diff
@@ -1115,7 +1120,7 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         if (ancestorPos_.depth() == SPLIT_DEPTH + N_BITS) {
             ancestorClaim_ = ancestorClaimRoot;
         } else {
-            ancestorClaim_ = getClaim(ancestorClaimRoot.raw(), _pos, _daItem);
+            ancestorClaim_ = getClaim(ancestorClaimRoot.raw(), Position.wrap(_pos.raw() - 1), _daItem);
```

Comment:
Should `_pos` be replaced by `ancestorPos_`?

Reply:
Actually, `ancestorPos_` should be the right context, but the `ancestorPos_` value is not right. Maybe we should rename `_pos` to `ancestorPos_`?